Fix bogus error (no idea how this is is supposed to work...):

  gmake: *** No rule to make target '-lxerces-c', needed by '...'.

Index: libxsd-frontend/build/import/libxerces-c/stub.make
--- libxsd-frontend/build/import/libxerces-c/stub.make.orig
+++ libxsd-frontend/build/import/libxerces-c/stub.make
@@ -12,7 +12,7 @@ ifdef libxerces_c_installed
 
 ifeq ($(libxerces_c_installed),y)
 
-$(call export,l: -lxerces-c,cpp-options: )
+#$(call export,l: -lxerces-c,cpp-options: )
 
 else
 
